assets/app/scripts/directives/logViewer.js

```diff
@@ -268,8 +278,42 @@ angular.module('openshiftConsole')
                   });
                 });
               });
+
+
+              var cachedNode;
+              var scrollableDOMNode;
+              var isWindow;
+              // scrollable node is window if mobile, else a particular DOM node.
+              var setScrollable = function() {
+                if(window.innerWidth < BREAKPOINTS.screenSmMin) {
+                  scrollableDOMNode = window;
+                  isWindow = true;
+                } else {
+                  scrollableDOMNode = cachedNode;
+                  isWindow = false;
+                }
+              };
+
+              // API to share w/link fn
+              this.cacheScollableNode = function(node) {
+                cachedNode = node;
+                setScrollable();
+              };
+              // maintain the correct scrollable node
+              $(window).on('resize', setScrollable);
```

Comment:
We can, I was attempting to keep the behavior a bit more separated, there is a lot going on in this directive.  I can push it up into the single listener though.